Remora.Commands/Signatures/ParameterShapes/PositionalCollectionParameterShape.cs

```diff
@@ -179,7 +228,7 @@ public override bool Matches
     /// <inheritdoc/>
     public override bool IsOmissible(TreeSearchOptions? searchOptions = null)
     {
-        if (this.Parameter.IsOptional)
+        if (this.IsOptional)
```

Comment:
Thinking out loud; the `IsOptional`/`DefaultValue` pair could potentially be replaced with an `Optional<T>` if we bring in Remora.Rest.Core.

Reply:
Your call, it'd be a trivial fix in theory, however I'm not sure the scope of such a change